articles/quickstart/native/react-native-expo/00-login.md

```diff
@@ -14,6 +14,10 @@ useCase: quickstart
 ---
 
 <!-- markdownlint-disable MD002 MD012 MD041 -->
+::: note
+Exciting news! We have just released our latest beta version of v3, packed with powerful features and improvements. [Check out the latest Quickstart for more details](https://auth0.com/docs/quickstart/native/react-native-expo-beta/00-login)
```

Comment:
I would tweak this a bit, just a suggestion. The "beta version of v3" reads a little awkwardly.

```suggestion
Exciting news! We have just released React Native Auth0 v3 BETA, packed with powerful features and improvements. [Check out the latest Quickstart for more details](https://auth0.com/docs/quickstart/native/react-native-expo-beta/00-login)
```

Whatever you change it to, I would change the others to match too.